src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -73,7 +73,7 @@ export class ProjectTokenHistoricalPriceService {
    * @returns Array(IHistoricalPriceRecord): {time: number, price?: number}
    */
   public async getPricesHistory(lbpMgr: LbpManager): Promise<Array<IHistoricalPriceRecord>> {
-    if (!lbpMgr.lbp || !lbpMgr.lbp.poolId || lbpMgr.startTime.getTime() >= new Date().getTime()) {
+    if (!lbpMgr.lbp || !lbpMgr.lbp.poolId || lbpMgr.hasNotStarted) {
```

Comment:
Is supposed to be `!lbpMgr.hasNotStarted`  (see the '!')

Reply:
hasNotStarted returns `true` 













;

Reply:
my brain is just too ready for a break